pdns/dnsdistdist/Makefile.am

```diff
@@ -166,7 +166,7 @@ dnsdist_SOURCES = \
 	dnsdist-lua-bindings.cc \
 	dnsdist-lua-ffi-interface.h dnsdist-lua-ffi-interface.inc \
 	dnsdist-lua-ffi.cc dnsdist-lua-ffi.hh \
-	dnsdist-lua-inspection-ffi.cc dnsdist-lua-inspection-ffi.hh \
+	dnsdist-lua-inspection-ffi.cc \
```

Comment:
As `dnsdist-lua-inspection-ffi.h` is included in `dnsdist-dynblocks.hh`, I'm wondering why it is left out of `dnsdist_SOURCES`. AFAIU it is needed for a build.

Reply:
Good one, thanks, I'll investigate!

Reply:
Good catch, we had to fix that in a follow-up PR on master and I managed to miss it. Fixed now!